ukvm/ukvm_gdb_kvm_x86_64.c

```diff
@@ -0,0 +1,435 @@
+/* 
+ * Copyright (c) 2015-2017 Contributors as noted in the AUTHORS file
+ *
+ * This file is part of ukvm, a unikernel monitor.
+ * 
```

Comment:
Nit: To keep the license headers standardised, can you please start the file with the standard header used everywhere else and move the "based on XXXX and its licenses" to a separate comment following the header.